internal/mode/static/telemetry/collector.go

```diff
@@ -315,3 +324,14 @@ func collectClusterInformation(ctx context.Context, k8sClient client.Reader) (cl
 
 	return clusterInfo, nil
 }
+
+func computeBackendTLSPolicyCount(policies map[types.NamespacedName]*graph.BackendTLSPolicy) (
+	backendTLSPolicyCount int64,
+) {
+	for _, policy := range policies {
+		if policy.Valid {
```

Comment:
Why check for valid here? I think we still want to report invalid policies cc @ciarams87

Reply:
I can't see why we wouldn't report invalid policies 🤔

Reply:
Me either. @salonichf5 can you remove this check?

Reply:
So, the `backendTLSPolicyCount` will just directly count the number of backend tls policies in that list?

Reply:
Well, you don't need the function at all. You can just set the field equal to the length of the backend tls policies list.